src/main/java/com/cloudogu/smeagol/repository/infrastructure/RepositoryResource.java

```diff
@@ -0,0 +1,48 @@
+package com.cloudogu.smeagol.repository.infrastructure;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.springframework.hateoas.ResourceSupport;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class RepositoryResource extends ResourceSupport {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS2160)